presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2179,8 +2179,8 @@ public void testCreateTableWithHeaderAndFooter()
                         ")\n" +
                         "WITH (\n" +
                         "   format = 'TEXTFILE',\n" +
-                        "   textfile_skip_footer_line_count = 1,\n" +
-                        "   textfile_skip_header_line_count = 1\n" +
+                        "   skip_footer_line_count = 1,\n" +
+                        "   skip_header_line_count = 1\n" +
```

Comment:
can you please add tests for csv with `skip_header_line_count` and `skip_footer_line_count` here?